docs-source/content/userguide/managing-domains/choosing-a-model/_index.md

```diff
@@ -1,23 +1,27 @@
-cloud providers that may make one approach better suited to your needs.
-You can also mix and match on a domain-by-domain basis.
+When using the operator to deploy a WebLogic domain, you have the choice of the following WebLogic domain home source types:
 
-| Domain on a persistent volume | Domain in a Docker image |
-| --- | --- |
-| Let's you use the same standard read-only Docker image for every server in every domain. | Requires a different image for each domain, but all servers in that domain use the same image. |
-| No state is kept in Docker images making them completely throw away (cattle not pets). | Runtime state should not be kept in the images, but applications and configuration are. |
-| The domain is long-lived, so you can mutate the configuration or deploy new applications using standard methods (Administration Console, WLST, and such). | If you want to mutate the domain configuration or deploy application updates, you must create a new image. |
-| Logs are automatically placed on persistent storage.  | Logs are kept in the images, and sent to the pod's log (stdout) unless you manually place them on persistent storage.  |
-| Patches can be applied by simply changing the image and rolling the domain.  | To apply patches, you must create a new domain-specific image and then roll the domain.  |
-| Many cloud providers do not provide persistent volumes that are shared across availability zones, so you may not be able to use a single persistent volume.  You may need to use some kind of volume replication technology or a clustered file system. | You do not have to worry about volume replication across availability zones since each pod has its own copy of the domain.  WebLogic replication will handle propagation of any online configuration changes.  |
-| CI/CD pipelines may be more complicated because you would probably need to run WLST against the live domain directory to effect changes.  | CI/CD pipelines are simpler because you can create the whole domain in the image and don't have to worry about a persistent copy of the domain.  |
-| There are less images to manage and store, which could provide significant storage and network savings.  |  There are more images to manage and store in this approach. |
-| You may be able to use standard Oracle-provided images or, at least, a very small number of self-built images, for example, with patches installed. | You may need to do more work to set up processes to build and maintain your images. |
+ - **Domain in PV**: Supply your domain home configuration in a persistent volume.
+ - **Domain in Image**: Supply your domain home in a Docker image.
+ - **Model in Image**: Supply a WebLogic Deployment Tool model file in a Docker image.
+
+There are advantages to all approaches, and there are sometimes technical limitations of various cloud providers that may make one approach better suited to your needs.  You can also mix and match on a domain-by-domain basis.
+
```

Comment:
Shall we explain how customers can mix-and-match those types? I assume they can only mix-and-match some sub-features, and we probably need to point those out.

Reply:
@rjeberhard  - My read is that this is simply  supposed to mean that you can have different types of domains in the same k8s cluster or even the same namespace.   Is that correct? Can you suggest a rewording?  Or directly make the change yourself?

Reply:
I was not sure what it was supposed to tell the customers. If that is the case as you mentioned, I suggest we remove the sentence.

Reply:
@doxiao @rjeberhard How about rewording to:   `There are advantages for each domain home source type, but sometimes there are technical limitations of various cloud providers that may make one type better suited to your needs.  Note that your are free to use different domain home types for different domains; there is no restriction on having domains with different domain types deployed to the same Kubernetes cluster or namespace.` ?

Reply:
Sounds good to me 👍

Reply:
Done! 

@doxiao Please close this comment if you're fine with the update.

Reply:
looks good to me.

Reply:
OK. resolving.

Reply:
Oh. it's already resolved. Ignore that.  Thanks Dongbo!